### PR TITLE
Saints updates

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: new/PRS13682Athanasius.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
```
Comment (MarcinKrawczuk) [inline new/PRS13682Athanasius.xml:36]: Keywords "Saint" and "Martyr"?

**Review by MarcinKrawczuk — APPROVED** [review]

File: PRS12001-13000/PRS12651Pishoy.xml [hunk 2]
```diff
@@ -48,11 +49,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                 <person sex="1" sameAs="wd:Q2696973">
                     <persName xml:lang="gez" xml:id="n1">ብሶይ፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Bǝsoy</persName>
-                    <persName xml:lang="en" xml:id="n2">Pishoy</persName>
-                    <faith type="Coptic"/>
-                    <occupation type="ecclesiastic"/>
+                    <persName xml:lang="en" corresp="#n1">Pishoy</persName>
+                    <persName xml:lang="gez" xml:id="n2">እብሶይ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">ʾǝbsoy</persName>
```
Comment (MarcinKrawczuk) [inline PRS12001-13000/PRS12651Pishoy.xml:55]: ʾƎbsoy?

File: new/PRS13684Baamin.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      </profileDesc>
+      <revisionDesc>
+         <change who="ES" when="2022-01-05">Created XML record</change>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:base="https://betamasaheft.eu/">
+      <body>
+         <listPerson>
+            <person sex="1">
+               <persName xml:lang="gez" xml:id="n1">በኪሞስ፡</persName>
```
Comment (MarcinKrawczuk) [inline new/PRS13684Baamin.xml:53]: Shouldn't it be በአሚን ?

File: new/PRS13680Arwa.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">አርዋ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾArwā</persName>
+                    <faith type="Judaism Christianity"/>
+                    <floruit>"Beautiful Jewish woman, who suffered much for being faithful to her husband while he was absent" <note><bibl><ptr target="bm:EMML5"/></bibl></note></floruit>
+                    <death>449, in Memphis. He is commemorated on <date type="diesNatalis" ref="ethiocal:Genbot30">30 Gǝnbot</date></death>
```
Comment (MarcinKrawczuk) [inline new/PRS13680Arwa.xml:57]: She

File: new/PRS13681Askanafer.xml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit><witness corresp="EMML1844">   EMML 1844 (XVI s.): f. 114v (19 Maggābit; menzione nel margine).
+                </witness>
```
Comment (MarcinKrawczuk) [inline new/PRS13681Askanafer.xml:25]: Do we really need witnesses here? Maybe keep them in the work record only.

File: PRS2001-3000/PRS2467Barsoma.xml [hunk 6]
```diff
@@ -32,26 +32,51 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  Around 1300 he retired to the Dayr Šahrān monastery south of Cairo. He died there on August 28, 1317 and was soon venerated as a saint by the faithful. </floruit>
+               <note type="commemoration"><date type="diesNatalis" ref="ethiocal:Yakkatit9">9 Yakkātit </date><date type="diesNatalis" ref="ethiocal:Pagwemen5">5 agʷmen</date></note>
+               <!-- Mark N. Swanson: The Life and the Miracles of Barsawma al-ʿUryan. In: David Thomas et al. (Hrsg.): Christian-Muslim Relations. A Bibliographical History. Bd. 5: 1350–1500. Brill, Leiden 2013, S. 114–118.
+Walter E. Crum: Barsaumâ The Naked. In: Proceedings of the Society of Biblical Archeology 29 (1907) 135–149. 187–206.
+ -->
             </person>
          </listPerson>
+         <listRelation>
+            <relation name="ecrm:P129i_is_subject_of" active="PRS2467Barsoma" passive="LIT4826GadlaBa"/>            
+            <relation name="ecrm:P129i_is_subject_of" active="PRS2467Barsoma" passive="LIT6541MirBarsoma"/>
```
Comment (MarcinKrawczuk) [inline PRS2001-3000/PRS2467Barsoma.xml:78]: Maybe add sth like "different:from" to distinguish him from the Syriac Barsoma?

File: PRS11001-12000/PRS11288AnAzMi.xml [hunk 7]
```diff
@@ -34,6 +34,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <textClass>
+            <keywords>
+               <term key="saint"/>
+               <term key="martyr"/>
```
Comment (MarcinKrawczuk) [inline PRS11001-12000/PRS11288AnAzMi.xml:40]: It's probably personal taste but I would use biblical.

File: PRS11001-12000/PRS11288AnAzMi.xml [hunk 8]
```diff
@@ -56,8 +62,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <persName xml:lang="gez" type="normalized" corresp="#n2">Salastu daqiq</persName>
```
Comment (MarcinKrawczuk) [inline PRS11001-12000/PRS11288AnAzMi.xml:62]: Śalastu

File: PRS11001-12000/PRS11288AnAzMi.xml [hunk 9]
```diff
@@ -66,6 +71,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             </personGrp>
             <listRelation>
                <relation name="snap:Group" active="PRS11288AnAzMi" passive="PRS11464Azaria PRS11465Anania PRS11463Misael"/>
+               <relation name="ecrm:P129i_is_subject_of" active="PRS11288AnAzMi" passive="LIT1566hababa"></relation>
+               <relation name="ecrm:P129i_is_subject_of" active="PRS11288AnAzMi" passive="LIT1664Homily"></relation>
```
Comment (MarcinKrawczuk) [inline PRS11001-12000/PRS11288AnAzMi.xml:75]: There are more e.g. LIT5656Malheq#ThreeYouth, LIT1828Mahale#ThreeYouths2, LIT1232Cantic...

File: PRS11001-12000/PRS11980Cornelius.xml [hunk 10]
```diff
@@ -50,16 +51,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Qornolyos</persName>
                     <persName xml:lang="gez" xml:id="n2">ቆርኔሌዎስ፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n2">Qornelewos</persName>
-                    <persName xml:lang="en" xml:id="n3" type="main">Cornelius <addName type="epithet">the Centurion</addName>
-               </persName>
-                    <persName xml:lang="en" xml:id="centurion">
-                  <addName type="epithet">the Centurion</addName>
+                    <persName xml:lang="gez" xml:id="n3">ቀርኔልዮስ፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n3">Qarnelyos </persName>
```
Comment (MarcinKrawczuk) [inline PRS11001-12000/PRS11980Cornelius.xml:55]: Qarnelǝyos?

File: new/PRS13681Askanafer.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/PRS13681Askanafer.xml:25]: As far as I understand, only for the work record, otherwise you can record it through "relations".

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
@DenisNosnitsin1970 can you approve please? thank you

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**